### PR TITLE
Create new .well class to use for boxes of content

### PR DESCRIPTION
**Pull request by jseppi**
...that were previously classed with "usa-alert usa-alert-info" because
they are not semantically alerts, and we don't want the iconography
associated with those alert classes.

Also, clean up some other alert mark-up and classes.

Here's how the site template wells look like now (same as in prod, but fixes issue #986):

screen shot 2017-06-07 at 5 42 43 pm

Here is how the wells look on the site settings page:

screen shot 2017-06-07 at 4 25 27 pm

Here is how the AlertBanners look:

screen shot 2017-06-07 at 5 44 51 pm

### CODE REVIEW

**Comment by wslack** [comment]
Also the linter is unhappy!

File: frontend/components/AddSite/TemplateSiteList/templateSite.js [hunk 1]
```diff
@@ -1,18 +1,18 @@
 import React from 'react';
 
+import { getSafeRepoName } from '../../../util';
+
 const propTypes = {
   name: React.PropTypes.string.isRequired,
-  owner: React.PropTypes.string.isRequired,
-  branch: React.PropTypes.string.isRequired,
-  repo: React.PropTypes.string.isRequired,
```
Comment (jseppi) [inline frontend/components/AddSite/TemplateSiteList/templateSite.js]: `branch` and `repo` seemed completely unused. Please confirm @jmhooper.

File: frontend/components/AddSite/TemplateSiteList/templateSite.js [hunk 2]
```diff
@@ -1,18 +1,18 @@
 import React from 'react';
 
+import { getSafeRepoName } from '../../../util';
+
 const propTypes = {
   name: React.PropTypes.string.isRequired,
-  owner: React.PropTypes.string.isRequired,
```
Comment (jseppi) [inline frontend/components/AddSite/TemplateSiteList/templateSite.js]: `owner` appeared unused, but I did see reference to using `props.defaultOwner`, so I renamed `owner` to `defaultOwner`

File: frontend/components/AddSite/TemplateSiteList/templateSite.js (hunk 1 above)
Comment (jmhooper) [inline frontend/components/AddSite/TemplateSiteList/templateSite.js]: They might be. I copy/pasted what was coming from the API.

File: frontend/components/AddSite/TemplateSiteList/templateSite.js [hunk 3]
```diff
@@ -30,56 +30,51 @@ class TemplateSite extends React.Component {
...
   }
 
-  handleChooseActive() {
-    this.props.handleChooseActive(this.props.index);
+  getFormVisible() {
+    const { active, index } = this.props;
+    return active === index;
   }
 
-  getSafeRepoName(name) {
```
Comment (jseppi) [inline frontend/components/AddSite/TemplateSiteList/templateSite.js]: this has been extracted to `utils/index.js`, and tests have been added.

File: frontend/components/AddSite/TemplateSiteList/templateSite.js (hunk 1 above)
Comment (jmhooper) [inline frontend/components/AddSite/TemplateSiteList/templateSite.js]: Wrong PR, nevermind me. I'll check on it

Comment (jmhooper) [inline frontend/components/AddSite/TemplateSiteList/templateSite.js]: Just checked. Those used to be used, but fell out of usage in commit 9b0b2e6554a61f14073dd51c34fa4f115648e154

**Pull request merged**